src/app/(auth)/base.css

```diff
@@ -66,4 +66,11 @@
 		/* Styling (Background) */
 		@apply bg-[#5A8472];
 	}
+	#reset-link-span,
```

Comment:
Unsure if this code is being used anywhere?

If not remove this block scope.

Reply:
Was this resolved Miguel ?

Reply:
This code is not needed. I am removing it.